docs/index.asciidoc

```diff
@@ -26,6 +26,15 @@ configuration item is `port`, which specifies the udp port logstash
 will listen on for event streams.
 
 
+==== Event Metadata and the Elastic Common Schema (ECS)
+
+This plugin adds a field containing the source IP address of the UDP packet. By default it goes
+in a field named `host` but when ECS is not disabled it's renamed to `[host][ip]`.
+When the name is customized, using the <<plugins-{type}s-{plugin}-source_ip_fieldname>> no ECS
+renaming kicks in, giving high priority to the user preference.
+For a complete listing of metadata shape, see <<plugins-{type}s-{plugin}-ecs_compatibility>>.
```

Comment:
// cc @karenzone for a review, think we should rephrase things such as the double negation: 
~~"ECS is not disabled"~~ -> "ECS is enabled".